django_extensions/management/commands/clean_pyc.py

```diff
@@ -12,7 +12,7 @@
 class Command(BaseCommand):
     help = "Removes all python bytecode compiled files from the project."
 
-    requires_system_checks = False
+    requires_system_checks = []
```

Comment:
in newer version of django, this value should be list/tuple of `Tags` or `__all__`. You can check the documentation [here](https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/#django.core.management.BaseCommand.requires_system_checks).

For `mypy` linting to work well with backward comptability, I think one of the solution would be to do something of the form:
```python

from typing import List, Tuple, Union

from django.core.checks import Tags

requires_system_checks: Union[bool, List[Tags], Tuple[Tags], str]
```

Instead of making this change everywhere in the codebase, I think it would be better to create some sort of `BaseCommand`and let all commands inherit from it. 

Would it be acceptable? Any suggestions are welcome.